### ISSUE TITLE
trying to set a read only property doesn't cause a compiler error

### ISSUE THREAD

**Issue by mareek** [Microsoft/TypeScript#6429]
If one try to set a property that hasn't any getter defined, the compiler doesn't emit any error or warning and the code silently fails à runtime.

example:
```typescript
class Greeter {
    private _greeting: string;
    get greeting(): string { return this._greeting; }

    constructor(message: string) {
        this.greeting = message; //no error at compile time and silent failure at execution time
    }
    greet() {
        return "Hello, " + this.greeting;
    }
}
```

**Comment by kitsonk** [comment]
Essentially a dupe of #295 (and other issues) but related to/would be fixable by #12 being resolved.

**Comment by RyanCavanaugh** [comment]
Please search for duplicates before logging issues.
